src/aspire/utils/numeric/scipy_fft.py

```diff
@@ -0,0 +1,14 @@
+import scipy as sp
```

Comment:
This is not a convention. Import fftpack explicitly. See https://docs.scipy.org/doc/scipy/reference/api.html#guidelines-for-importing-functions-from-scipy

Reply:
Changed in commit e83b287.